classes/Pods.php

```diff
@@ -1015,12 +1015,25 @@ public function field( $name, $single = null, $raw = false ) {
 						$field_names = explode( '.', $params->name );
 
 						if ( isset( $field_names[1] ) ) {
-							$size = $field_names[1];
+							$size    = $field_names[1];
+							$sizes   = get_intermediate_image_sizes();
+							// Not shown by default.
+							$sizes[] = 'full';
+							$sizes[] = 'original';
+							if ( ! in_array( $size, $sizes, true ) ) {
+								// No valid image size found.
+								$size = false;
+							}
 						}
 					}
 
-					// Pods will auto-get the thumbnail ID if this isn't an attachment.
-					$value = pods_image( $this->id(), $size, 0, null, true );
+					if ( $size ) {
+						// Pods will auto-get the thumbnail ID if this isn't an attachment.
+						$value = pods_image( $this->id(), $size, 0, null, true );
+					} else {
+						array_shift( $field_names );
+						$value = pods( 'media', get_post_thumbnail_id( $this->id() ) )->field( implode( '.', $field_names ));
```

Comment:
* :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: 23ef1d03fdfed9639c9823e0cfe89345</sub>